packages/utils/constants/TripTypes.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+export type TripTypes = 'Return' | 'OneWay' | 'MultiCity';
```

Comment:
I was just thinking if `@kiwicom/margarita-config` package will not be more suitable for this.
There is quite thin border between those two packages.
But as I see it now, this package is probably more suitable for constants like this.
I'm just going to create new issue for `utils` folder structure revision, as you probably noticed while adding this, it is not really unified.

Reply:
Yes! also there is a babel error when trying to import and use `margarita-utils` on the `graphql` package. It is not transpiled (`margarita-config` is as it uses `modules.export`)

Reply:
The issue is not wether it uses modules.exports or not. Nothing should use modules.export, it is a sign that something is wrong, and we should fix it rather than making temporary patches that will give us further problems in the future. 

Luckily we already solved this in `incubator/universe`, so I will prepare a PR to fix this.

Reply:
after #621 is merged, it should work just fine.

Reply:
Moved to `margarita-config` to overcome this problem!